R/application.R

```diff
@@ -0,0 +1,49 @@
+#' @title Application
+#' @docType class
+#' @description  Application representing one instance of an application in a Simulation
+#'
+#' @format NULL
+Application <- R6::R6Class(

```

Comment:
@pchelle @abdullahhamadeh @Yuri05 
This is one instance of an application in the model
Parameters are
startTime, infusion, drugmass

starttime and drugmass should always be present. Infusion time maybe

@pchelle in your case you only need start time

This is a parameter object so to get the value you have to do app$starTime$value